code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -156,43 +156,162 @@
 
-/datum/emote/living/carbon/human/monkey/roll
+/datum/emote/living/carbon/human/roll
 	key = "roll"
 	key_third_person = "rolls"
 	message = "rolls."
 	hands_use_check = TRUE
+	species_type_whitelist_typecache = list(/datum/species/monkey)
 
-/datum/emote/living/carbon/human/monkey/scratch
+/datum/emote/living/carbon/human/scratch
 	key = "scratch"
 	key_third_person = "scratches"
 	message = "scratches."
 	hands_use_check = TRUE
+	species_type_whitelist_typecache = list(/datum/species/monkey)
 
-/datum/emote/living/carbon/human/monkey/screech/roar
+/datum/emote/living/carbon/human/roar
 	key = "roar"
 	key_third_person = "roars"
 	message = "roars."
 	emote_type = EMOTE_AUDIBLE
+	species_type_whitelist_typecache = list(/datum/species/monkey)
 
-/datum/emote/living/carbon/human/monkey/tail
+/datum/emote/living/carbon/human/tail
 	key = "tail"
 	message = "waves their tail."
+	species_type_whitelist_typecache = list(/datum/species/monkey)
 
 /datum/emote/living/carbon/human/monkeysign
 	key = "sign"
 	key_third_person = "signs"
 	message_param = "signs the number %t."
 	hands_use_check = TRUE
+	species_type_whitelist_typecache = list(/datum/species/monkey)
+
+
+//Moth emotes
+/datum/emote/living/mothsqueak
+	key = "msqueak"
+	key_third_person = "lets out a tiny squeak"
+	message = "lets out a tiny squeak!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'sound/voice/mothsqueak.ogg'
+	species_type_whitelist_typecache = list(/datum/species/moth)
+
+/datum/emote/living/chitter
+	key = "chitter"
+	key_third_person = "chitters"
+	message = "chitters!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'sound/emotes/mothchitter.ogg'
+	species_type_whitelist_typecache = list(/datum/species/moth)
+
+//Felinid emotes
+/datum/emote/living/meow
+	key = "meow"
+	key_third_person = "meows"
+	message = "meows!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'sound/emotes/meow.ogg'
+	species_type_whitelist_typecache = list(/datum/species/human/felinid)
+
+/datum/emote/living/purr //Ported from CitRP originally by buffyuwu.
+	key = "purr"
+	key_third_person = "purrs!"
+	message = "purrs!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'sound/voice/feline_purr.ogg'
+	species_type_whitelist_typecache = list(/datum/species/human/felinid)
+
+//Shared custody between felinids and lizards
+/datum/emote/living/hiss
+	key = "hiss"
+	key_third_person = "hisses"
+	message = "hisses!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'sound/emotes/hiss.ogg'
+	mob_type_allowed_typecache = list(/mob/living/carbon, /mob/living/silicon/pai)
+	species_type_whitelist_typecache = list(/datum/species/lizard, /datum/species/human/felinid)
+
+//Lizard emotes
+/datum/emote/living/rpurr
+	key = "rpurr"
+	key_third_person = "purrs!"
+	message = "purrs!"
+	emote_type = EMOTE_AUDIBLE
+	vary = TRUE
+	sound = 'sound/voice/raptor_purr.ogg'
+	species_type_whitelist_typecache = list(/datum/species/lizard)
+
+//Skrell emotes
+/datum/emote/living/carbon/human/warble
```

Comment:
Share with teshari. Share concerns against with Teshari lore maintainer.

Reply:
I have no means to restrict it to teshari considering https://github.com/pariahstation/Pariah-Station/pull/688 is not yet merged.